examples/analog_out/cont_gen_voltage_wfm_int_clk.py

```diff
@@ -0,0 +1,32 @@
+"""Example of analog output voltage generation.
+
+This example demonstrates how to output a continuous periodic
+waveform using an internal sample clock.
+"""
+
+import numpy as np
+
+import nidaqmx
+from nidaqmx.constants import AcquisitionType
+
+with nidaqmx.Task() as task:
+    frequency = 10
```

Comment:
FYI, the ANSI C examples use 9.95 because some older devices like E Series and some S Series can't output full-scale voltage without coercing and returning a warning. (This is because the max voltage is 1 LSB smaller than the min voltage: 32767 vs. -32768, etc.)

Reply:
Noted. If that's the case, should I update the frequence value to 9.95 to support these older devices?